src/kontrol/cli.py

```diff
@@ -357,6 +357,13 @@ def parse(s: str) -> list[T]:
         action='store_true',
         help='Include auxiliary Kontrol lemmas.',
     )
+    build.add_argument(
+        '--evm-rule-optimizations',
+        dest='evm_rule_optimizations',
+        default=None,
+        action='store_true',
+        help='Include optimized KEVM rules.',
```

Comment:
Maybe a more detailed help message?
```suggestion
        help='Include optimized KEVM rules. Uses the assumption that the EVM bytecode
that comes compiled from Solidity will not result in word stack overflows or underflows.',
```